src/pages/profile/VerifyAirbnb.js

```diff
@@ -107,28 +130,49 @@ class VerifyAirbnb extends Component {
   renderShowGeneratedCode(){
     return(
       <form
-        onSubmit={async e => {
-          e.preventDefault()
-          try {
-            let airbnbAttestation = await origin.attestations.airbnbVerify({
+        onSubmit={async event => {
+          await this.catchPossibleErrors(async event => {
+            event.preventDefault()
+            this.clearErrors()
+
+            const airbnbAttestation = await origin.attestations.airbnbVerify({
               wallet: this.props.wallet,
               airbnbUserId: this.getUserIdFromAirbnbProfile(this.state.airbnbProfile)
             })
 
             this.props.onSuccess(airbnbAttestation)
-          } catch(e){
-            let unknownError = <FormattedMessage
-              id={ 'VerifyAirbnb.unknownError' }
-              defaultMessage={ 'An unknown error occurred' }
-            />
-
-            let error = JSON.parse(e)
-            error = error.errors ? error.errors.join('</br>') : unknownError
-            this.setState({ error: error })
-          }
+          }, event)
         }}
       >
-        {this.renderGeneratedCode()}
+        <div className="form-group">
+          <label htmlFor="airbnbProfile">
+            { <FormattedMessage
+              id={ 'VerifyAirbnb.enterCodeIntoAirbnb' }
+              defaultMessage={ 'Go to Airbnb website, edit your profile and paste the following text into profile description.' }
+            /> }
+          </label>
+          <textarea
+            className="form-control"
+            id="generated-code"
+            readOnly="readOnly"
+            // Making input wider, so that the whole verification code can be viewed without trimming.
+            style={{maxWidth: '340px', height: '92px', resize: 'none'}}
+            value={
+              this.state.confirmationCode == '' ?
+                this.props.intl.formatMessage({
+                  id: 'VerifyAirbnb.loadingConfirmationCode',
+                  defaultMessage: 'Loading...'
+                }) :
+                "Origin verification code: " + this.state.confirmationCode
```

Comment:
I see that this was also already in the code and we don't have a linter to enforce this yet, but I think we're trying to use template literals instead of string concatenation (it won't let me write backticks because of markdown but you get the idea):
`Origin verification code: ${this.state.confirmationCode}`